src/main.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'vue-unicons' {
+  export function install(Vue: any): void
```

Comment:
I know very little about TS, but I heard that using the `any` type is an anti-pattern. Vue also has type declarations.

Reply:
My mistake, `export function install(Vue: VueConstructor, options?: object): void;` would be better for that line.

It adds the `options` argument as well but doesn't make it required, since I notice the `.install` method doesn't actually refer to the options argument at all at this time.

If you need me to submit a new pull request then let me know, I'm new to the whole process on GitHub.

Reply:
Create a new PR is not necessary. You just need to commit all changes and PR itself to update.